### PR TITLE
BookA-Pset_SpatialGeoObsExcavatedZone

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_SpatialGeoObsExcavatedZone

## Description: 

Common properties for excavated zone observations

| Property | Datatype | Description |
|----------|----------|-------------|
|TunnelExcavationMethod|IfcLabel|Type of tunnel excavation method that was applied when the observation was made|
|ExcavationFaceType|PEnum_ExcavationFaceType|The excavation surface that was mapped.|
|ExcavationStep|PEnum_ExcavationStep|Excavation Step|
|ExcavationMethod|PEnum_ExcavationMethod|Excavation methods - change to Penum_UndergroundExcavationMethod?|
|LogNumber|IfcInteger|Identifer of the records|

## Enumeration: PEnum_ExcavationFaceType

| Constant | Description |
|----------|-------------|
|TUNNELFACE|Observation done on the face of the excavation.|
|TUNNELWALL|Observation done on the sides of the excavation. |
|TUNNELROOF|Observation done on the roof of the excavation. |
|INVERT|Observation done on the floor of the excavation. |

## Enumeration: PEnum_ExcavationStep

| Constant | Description |
|----------|-------------|
|TOPHEADING| |

## Enumeration: PEnum_ExcavationMethod

| Constant | Description |
|----------|-------------|
|DRILLBLAST| Should use PEnum_UndergroundExcavationMethod from PR #570 |

### CODE REVIEW

File: IFC4x3/Properties/e/ExcavationFaceType_2a9lnvIGXBJf$bF3uy85d9/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The part of an excavated space that was mapped.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/e/ExcavationFaceType_2a9lnvIGXBJf$bF3uy85d9/Documentation.md]: Space or face that was mapped?

File: IFC4x3/Properties/l/LogNo_2H$tb9GtfCP9DRUp61IODv/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LogNo_2H$tb9GtfCP9DRUp61IODv" Name="LogNo" UniqueId="91ff7949-437a-4c64-935b-7b3181498379" PrimaryDataType="IfcInteger" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/l/LogNo_2H$tb9GtfCP9DRUp61IODv/DocProperty.xml]: I would prefer a more descriptive name but I guess it is fine if the stakeholders say so.

File: IFC4x3/Properties/t/TunnelExcMethod_0IxY668fj68P9Wr33SXafb/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Type of the tunnel construction method that was applied when the mapping was performed.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/TunnelExcMethod_0IxY668fj68P9Wr33SXafb/Documentation.md]: What does it mean that "mapping was performed"?

**Review by SergejMuhic — COMMENTED** [review]
- Descriptions are a bit short.
- Property names have abbreviations.
- Descriptions of constants and enums missing.

File: IFC4x3/Properties/e/ExcavationFaceType_2a9lnvIGXBJf$bF3uy85d9/Documentation.md (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/e/ExcavationFaceType_2a9lnvIGXBJf$bF3uy85d9/Documentation.md]: "The excavation surface that was mapped." - fixed in [ebc851b](https://github.com/bSI-InfraRoom/IFC-Specification/pull/589/commits/ebc851b49d47f6426cb687367b124ed31c9053cf)

File: IFC4x3/Properties/l/LogNo_2H$tb9GtfCP9DRUp61IODv/DocProperty.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Properties/l/LogNo_2H$tb9GtfCP9DRUp61IODv/DocProperty.xml]: Changed to LogNumber in [ebc851b](https://github.com/bSI-InfraRoom/IFC-Specification/pull/589/commits/ebc851b49d47f6426cb687367b124ed31c9053cf)

File: IFC4x3/Properties/t/TunnelExcMethod_0IxY668fj68P9Wr33SXafb/Documentation.md (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/t/TunnelExcMethod_0IxY668fj68P9Wr33SXafb/Documentation.md]: Changed to "Type of tunnel excavation method that was applied when the observation was made" in [ebc851b](https://github.com/bSI-InfraRoom/IFC-Specification/pull/589/commits/ebc851b49d47f6426cb687367b124ed31c9053cf). Also changed name to TunnelExcavationMethod. Still to do: Reuse property UndergroundExcavationMethod #570 when merged.

File: IFC4x3/Properties/t/TunnelExcavationMethod_0IxY668fj68P9Wr33SXafb/Documentation.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Type of tunnel excavation method that was applied when the observation was made.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/TunnelExcavationMethod_0IxY668fj68P9Wr33SXafb/Documentation.md]:
```suggestion
Type of the tunnel excavation method that was applied when the observation was made.
```

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**